tcms/rpc/tests/test_testexecution.py

```diff
@@ -291,13 +291,45 @@ def test_update_with_single_caserun(self):
         self.assertEqual(tcr['status'], self.status_positive.name)
         self.assertEqual(tcr['sortkey'], 90)
 
+    def test_update_with_assignee_id(self):
+        tcr = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
+            "assignee": self.user.pk
+        })
+        self.assertEqual(tcr['assignee'], self.user.username)
+
+    def test_update_with_assignee_email(self):
+        tcr = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
```

Comment:
Before manipulating the assignee field tests should ensure that it is different from the target value. A simple assertNotEqual will do.

Note: technically not necessary b/c TestExecutionFactory will create a new user if not provided, but we don't want to rely blindly on this and it's better to explicitly check preconditions.